cupoch_conversions/CMakeLists.txt

```diff
@@ -23,61 +24,62 @@ message("zs: EIGEN3_INCLUDE_DIRS = " ${EIGEN3_INCLUDE_DIRS})
 GA_CHECK_CUDA()
 
 include_directories(
-    include
-    ${CUPOCH_INCLUDE_DIRS}
+        include
+        ${CUPOCH_INCLUDE_DIRS}
 )
 
 install(DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION include/${PROJECT_NAME}/
-)
+        DESTINATION include/${PROJECT_NAME}/
+        )
 
 install(DIRECTORY res
-  DESTINATION lib/${PROJECT_NAME}/
-)
-
-set_directory_properties(PROPERTIES COMPILE_DEFINITIONS "")
+        DESTINATION lib/${PROJECT_NAME}/
+        )
 
 APPEND_TARGET_ARCH_FLAGS()
 
+set_directory_properties(PROPERTIES COMPILE_DEFINITIONS "")
+APPEND_TARGET_ARCH_FLAGS()
 set(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS}
-    ${CUPOCH_NVCC_FLAGS}
-)
+        ${CUPOCH_NVCC_FLAGS}
+        )
 add_definitions(${CUPOCH_DEFINITIONS})
 message("zs: ${CMAKE_PROJECT_NAME} CUDA_NVCC_FLAGS = " ${CUDA_NVCC_FLAGS})
+include(GATest)
```

Comment:
add this into `cupoch_conversions/cmake/ga_build_common.cmake`  line 107 rather than here.